src/components/Router/StepComponentMap.ts

```diff
@@ -120,30 +124,54 @@ const buildCaptureStepComponents = (
   const documentStep = findStep('document')
 
   const complete = mobileFlow ? [ClientSuccess] : [Complete]
+  const captureStepTypes = new Set(['document', 'poa', 'face'])
+  const firstCaptureStepType = steps.filter((step) =>
+    captureStepTypes.has(step?.type)
+  )[0]?.type
+  const showCrossDeviceClientIntroForFaceStep =
+    mobileFlow && firstCaptureStepType === 'face'
+  const CrossDeviceClientIntroForFaceStep = showCrossDeviceClientIntroForFaceStep
+    ? [CrossDeviceClientIntro]
+    : []
+  const showCrossDeviceClientIntroForDocumentStep =
+    mobileFlow && firstCaptureStepType === 'document'
+  const CrossDeviceClientIntroForDocumentStep = showCrossDeviceClientIntroForFaceStep
+    ? [CrossDeviceClientIntro]
+    : []
+  const showCrossDeviceClientIntroForPoaStep =
+    mobileFlow && firstCaptureStepType === 'poa'
+  const CrossDeviceClientIntroForPoaStep = showCrossDeviceClientIntroForFaceStep
+    ? [CrossDeviceClientIntro]
+    : []
```

Comment:
I'm starting to think that the `crossDeviceClientIntro` should not be in the component array for any of the steps. I think you should display it conditionally if it's `mobileFlow` and if it has not been shown yet. You could use a flag in the redux store for that and use it in the `CrossDeviceMobileRouter` component.

Reply:
Wouldn't that still need the screen to be added in in the `StepComponentMap`? Or were you thinking something like adding it in as an additional step type in the `CrossDeviceMobileRouter` base on what's state the flag is in the redux store?

Reply:
> Or were you thinking something like adding it in as an additional step type in the CrossDeviceMobileRouter base on what's state the flag is in the redux store

Yes, you might not even need to use redux and it could be handled by a state in `CrossDeviceMobileRouter`